tests/strings/strings.json

```diff
@@ -1198,8 +1185,8 @@
             }, 
             {
                 "_PTR_C_CXX_index": 6, 
-                "_PTR_F_C_index": 28, 
-                "_function_index": 29, 
+                "_PTR_F_C_index": 30, 
+                "_function_index": 31, 
```

Comment:
This pair of indices is off by one, but previous ones above were the same.  Is that expected?

Are they just different indices that happened to coincide above?

Reply:
The generated bufferify functions are inserted into the list after the function it is copied from.  That causes the indexes to gradually get offset.  But it keeps them together in the generated source.